_includes/_global_js.html

```diff
@@ -0,0 +1,55 @@
+<!-- This should be inserted in the page *before* any page-specific JS -->
+<script src="/assets/js/jquery.js"></script>
+<script src="/assets/js/bootstrap.min.js"></script>
+{% for js_name in page.extra_js %}<script src="/assets/js/{{ js_name }}"></script>{% endfor %}
```

Comment:
Any reason we shouldn't just include the `extra_js` files from the project page in the `base.html` layout?